h5c/vic/src/main/webapp/plugin.xml

```diff
@@ -197,4 +197,96 @@
             </componentClass>
         </object>
     </extension>
-</plugin>
\ No newline at end of file
+
+    <extension id="com.vmware.vic.actionSet">
+      <extendedPoint>vise.actions.sets</extendedPoint>
+      <object>
+         <actions>
+            <!-- Create VCH -->
+            <com.vmware.actionsfw.ActionSpec>
+               <uid>com.vmware.vic.createVch</uid>
+               <label>New Virtual Container Host...</label>
+               <delegate>
+                  <className>com.vmware.vsphere.client.htmlbridge.HtmlActionDelegate</className>
+                  <object><root>
+                      <!-- execute the action on client-side (html view in a modal dialog) -->
+                      <actionUrl>/vsphere-client/vic/resources/build-dev/index.html?view=ui-actions</actionUrl>
```

Comment:
Currently we have different output folders when compiling in dev mode and prod mode. `build-dev` for the dev build and `dist` for prod build. This means this code will not work when built in the prod mode. I think it this is the right time to consolidate our output folders to `dist` to avoid problems like this in the future.